documentation.yml

```diff
@@ -7,9 +7,9 @@ toc:
       - getPendingWaiterState
       - hasPendingWaiters
       - _reset
-  - name: Test Waiter class
+  - name: Async Test Waiter class
```

Comment:
I don't want to reference the "class" at all here, I think just calling it `Test Waiter` would be perfect...

Reply:
Yep, seems fine.

Reply:
It's gone now, as we're explicitly documenting public exports now.